services/viva-ms/lambdas/generateRecurringCaseHtml.js

```diff
@@ -29,6 +29,36 @@ handlebars.registerHelper({
   },
 });
 
+async function getClosedUserCases(primaryKey) {
+  const dynamoDbQueryCasesParams = {
+    TableName: config.cases.tableName,
+    KeyConditionExpression: 'PK = :pk and begins_with(SK, :sk)',
+    FilterExpression: 'begins_with(#status.#type, :statusTypeClosed)',
+    ExpressionAttributeNames: {
+      '#status': 'status',
+      '#type': 'type',
+    },
+    ExpressionAttributeValues: {
+      ':pk': primaryKey,
+      ':sk': 'CASE#',
+      ':statusTypeClosed': 'closed',
+    },
+  };
+  return await to(dynamoDb.call('query', dynamoDbQueryCasesParams));
+}
+
+function setChangedCaseAnswerValues(currentAnswerList, previousAnswerList) {
+  for (const answer of currentAnswerList) {
```

Comment:
Code below **might** (not tested) do the same thing without modifying the input parameters

```js
function getChangedCaseAnswerValues(currentAnswerList, previousAnswerList) {
const updatedAnswers = currentAnswerList.map((answer) => {
  const previousAnswer = previousAnswerList.find(item => item.field.id === answer.field.id);
  
  if (previousAnswer?.value !== answer.value) {
    answer.field.tags.push('changed');
  }

  return answer
});

return updatedAnswers;
}
```

Reply:
fixed